### PR TITLE
Fixes issue24

### PR DESCRIPTION
**Pull request by mmmyr**
When calling loadBackup with no backup branch found, ask user for a backup branch name

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -320,12 +320,36 @@ class GitBackupSync {
 			return false;
 		}
 
+		let backupBranchName: string;
+		let branchInfo = await this._branchInfo.get(this._config.branchInfoPath, currentBranchName);
+		if (branchInfo === undefined) {
```
Comment (maxyu1115) [inline src/extension.ts:325]: I think it's not just when the branch info file isn't present; there's also the case where the branch info file doesn't contain the newest information. So a branch info file may be present, but the branch might not be specified in the branch info file

Comment (mmmyr) [reply src/extension.ts:325]: Sorry, I don't understand the problem here lol. I think as long as the current branch information does not exist in the branch info file, the value of branchInfo will be undefined. Are there any other possibilities?

Comment (maxyu1115) [reply src/extension.ts:326]: oops sry nvm, misread

**Review by maxyu1115 — APPROVED** [review]
Great job! (btw when merging, remember to choose "squash and merge", where you write a commit message that describes what this commit does)

**Pull request merged**